WooCommerce/Classes/POS/Models/POSProduct.swift

```diff
@@ -1,19 +1,28 @@
 import Foundation
+import class WooFoundation.CurrencyFormatter
 
 public struct POSProduct {
     public let itemID: UUID
     public let productID: Int64
     public let name: String
     public let price: String
+    public let currency: String
```

Comment:
> I thought it'd be easier if we can convert Yosemite.Product with CurrencyFormatter (that depends on the currency settings) to what's ready to be displayed in the UI instead of a separate function priceWithCurrency with the CurrencyFormatter dependency.

I was thinking a similar approach, in the way of converting from `Yosemite.Product` to `POSProduct` we can just decorate the object with any currency or other formatters we need to fill out the POS-specific properties. At the moment we can do this conversion directly in the factory while the data layer is implemented, but ideally we'd inject it through the entry point for testability.